src/main/resources/logback.xml

```diff
@@ -26,18 +26,20 @@
 
     <appender name="SYSLOG" class="ch.qos.logback.classic.net.SyslogAppender">
         <syslogHost>localhost</syslogHost>
-        <facility>AUTH</facility>
-        <suffixPattern>[%thread] %logger %msg</suffixPattern>
+        <facility>logaudit</facility>
+        <suffixPattern>[%level] [%d{HH:mm:ss.SSS}] [%thread] %logger{36} - %msg%n</suffixPattern>
```

Comment:
For audit logs, I'm pretty sure we are using something very much like that. For app-level logging, we generally have a very different set of fields to log (class name, thread name, log level, etc.)